.github/dependabot.yml

```diff
@@ -1,17 +1,27 @@
 version: 2
 updates:
   # Maintain dependencies for GitHub Actions
-  - package-ecosystem: 'github-actions'
+  - package-ecosystem: "github-actions"
     directory: '/'
     schedule:
-      interval: 'daily'
-    labels:
-      - 'gh actions dependencies'
+      # Check for updates on the first day of every month
+      interval: "monthly"
```

Comment:
```suggestion
      interval: "weekly"
```

Reply:
Let's go with reduction to weekly updates first to see how it works out. We can further reduce the interval later on.